dolphinscheduler-master/src/main/resources/application.yaml

```diff
@@ -135,8 +135,7 @@ spring:
     activate:
       on-profile: mysql
   datasource:
-    driver-class-name: com.mysql.jdbc.Driver
-    url: jdbc:mysql://127.0.0.1:3306/dolphinscheduler?useUnicode=true&characterEncoding=UTF-8
```

Comment:
why should we delete attribute `url`?

Reply:
Activate profile only define the data source driver, and other fields, such as URL, username, and password we define at another place. You can see [this](https://github.com/apache/dolphinscheduler/issues/10103#issuecomment-1132669903) and [this](https://github.com/apache/dolphinscheduler/pull/10128/files)

Reply:
Got it, Thanks for the clarification